docs/admin/project.md

```diff
@@ -0,0 +1,84 @@
+---
+description: Guide on how to manage projects and quotas as a Hopsworks administrator
+---
+
+# Manage Projects
+
+Hopsworks provides administrator with a view of the projects in a Hopsworks deployment.
+
+Hopsworks administrators are not automatically members of all the projects in a deployment however they can see which projects exist, who is their owners and they can limit the amount of storage and compute quotas each project has available. 
```

Comment:
A Hopsworks administrator is not automatically a member of all the projects in a cluster. However, they can see which projects exist, who is the project owner, and they can limit the storage quota and compute quota for each project.